src/applications/vaos/tests/mocks/v0.js

```diff
@@ -23,7 +23,7 @@ export function getVAAppointmentMock() {
       clinicFriendlyName: 'Fake',
       clinicId: 'fake',
       facilityId: 'fake',
-      sta6aid: 'fake',
+      sta6aid: null,
```

Comment:
This was causing some issues where we weren't falling back to `facilityId` appropriately